components/src/organisms/FixtureOption/index.tsx

```diff
@@ -35,7 +33,6 @@ export function FixtureOption(props: FixtureOptionProps): JSX.Element {
         padding={SPACING.spacing16 + ' ' + SPACING.spacing24}
```

Comment:
not related your change but can you modify this?
```suggestion
        padding={`${SPACING.spacing16} ${SPACING.spacing24}`}
```